packages/components/psammead-most-read/src/List/index.jsx

```diff
@@ -1,27 +1,30 @@
 import React from 'react';
 import styled from 'styled-components';
-import { oneOf, number, node, bool } from 'prop-types';
+import { oneOf, number, node } from 'prop-types';
 import {
-  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
   GEL_GROUP_5_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 import Grid from '@bbc/psammead-grid';
 import { mostReadListGridProps } from '../utilities/gridProps';
 
-const TwoColumnGrid = styled(Grid).attrs({
+const OneColumnGrid = styled(Grid).attrs({
   role: 'list',
 })`
   list-style-type: none;
   margin: 0;
   padding: 0;
   grid-auto-flow: column;
   grid-template-rows: repeat(
-    ${props => Math.ceil(props.numberOfItems / 2)},
+    ${props => props.numberOfItems},
     [col-start] auto [col-end]
```

Comment:
This will be my poor Css knowledge but just trying to understand why `[col-start]` and `[col-end]` are needed? Could just be `auto`?

Reply:
I think you're right. The convention of `[col-start]` and `[col-end]` is used to name grid lines. However, I don't see anywhere in psammead or simorgh where these line names are used so I'll remove them. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Layout_using_Named_Grid_Lines